doc/threat_model/Untrusted_Capsule_Isolation.md

```diff
@@ -0,0 +1,23 @@
+Untrusted Capsule Isolation
+===========================
+
+## Isolation Mechanism
+
+Untrusted capsules are limited to what they should be able to access within
+Rust's type system without using `unsafe`. That isolation is implemented by
+banning `unsafe` from use in untrusted capsule code. This isolation is
+vulnerable to code that exploits compiler bugs or bugs in `unsafe` code in
+libraries. When a board integrator chooses to use an untrusted capsule, they are
+responsible for auditing the code of the untrusted capsule to detect potentially
+malicious behavior. This relies in part on Rust's resistance to underhanded
```

Comment:
What is "this"?

Reply:
I clarified it. I was referring to the use of Rust's type system as a security isolation mechanism.